OWNERS

```diff
@@ -22,3 +22,10 @@ filters:
   "^ansible-ipi-install/.*":
     labels:
       - kind/installer
+
+  "^features":
+    reviewers:
+      - code-reviewers
+      - cnf-reviewers
+    approvers:
```

Comment:
this won't work, that's why we have https://github.com/openshift-kni/baremetal-deploy/blob/master/features/OWNERS, see https://github.com/openshift-kni/baremetal-deploy/pull/64

Reply:
Hi Mark, should we add them for the global filter then?

We want to limit bugging others when putting changes on docs and the ansible script folders as almost 99% of the time, prow assigned 'others' which are not relevant to the repo.

Any advice here?

Reply:
keep the aliases file in the root directory, but put reviewers and approvers into  /features/OWNERS... that *should* work

Reply:
or reviewers with filter and approvers in its own file, as you did in the new PR, should also work